cloudformation/secure-contact.template.yaml

```diff
@@ -0,0 +1,131 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Run the SecureContact service in AWS
+
+Parameters:
+  VpcId:
+    Description: The VPC in which SecureContact will run
+    Type: AWS::EC2::VPC::Id
+
+  AMI:
+    Description: Base AMI for SecureContact instances
+    Type: AWS::EC2::Image::Id
+
+  SSHAccessCidr:
+    Type: String
+    Description: A CIDR from which SSH access to the instance is allowed
+    AllowedPattern: ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}/[0-9]{1,2}$
+    ConstraintDescription: Parameter should be a CIDR block e.g. "1.2.3.4/32"
+
+  KeyName:
+    Description: An ssh keypair to put on the instance
+    Type: AWS::EC2::KeyPair::KeyName
+
+  AccountId:
```

Comment:
This can be done with a pseudo parameter I think?

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html